releng/setup-env.sh

```diff
@@ -738,6 +738,7 @@ if ! grep -Eq "^$toolchain_version\$" "$FRIDA_TOOLROOT/.version" 2>/dev/null; th
   else
     echo "Downloading and deploying toolchain..."
     $download_command "https://build.frida.re/toolchain-${toolchain_version}-${build_platform}-${build_arch}.tar.bz2" | tar -C "$FRIDA_TOOLROOT" -xj $tar_stdin || exit 1
+    find "$FRIDA_TOOLROOT/bin" -type f -exec sed -i '' -e "s#/opt/python27-64/bin/python#$PYTHON3#g" {} + > /dev/null 2>&1
```

Comment:
- The in-place option for `sed` is a bit different between GNU and BSD. I remember getting bitten by this in the past. Did you try this on both macOS and Linux?
- Is the stdio redirection necessary here?
- We should probably add a fallback that uses `which python3` when `PYTHON3` is not defined, to still make it possible to run this script standalone without too much fuzz.

Reply:
These are all good points. I have not tested on MacOS, but Linux I have tested on regularly. 

The stdio redirection is by no mean necessary, but it prevents an error line when it complete the find list.

And that fallback is a good idea. It has bee implemented.

Reply:
This is the message that it prevents when you hit the end of the find list
```
sed: can't read : No such file or directory
```

Reply:
Did you try replacing ` + > /dev/null 2>&1` with `\;`?